intermediate-algorithm-scripting-yahtzee-game/index.html

```diff
@@ -1,104 +1,126 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Simplified Yahtzee Dice Game</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
 
-<head>
-  <meta charset="utf-8" />
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Simplified Yahtzee Dice Game</title>
-  <link rel="stylesheet" href="css/style.css">
-  <script defer src="js/index.js"></script>
-</head>
+  <body>
+    <header>
+      <h1>Yahtzee!</h1>
+    </header>
 
-<body>
-  <header>
-    <h2>Welcome to a simplified version of Yahtzee. Amazon's #1 best-selling dice game.</h2>
-
-    <p>
-      <a href="https://www.wikihow.com/Play-Yahtzee" target="_blank">
-        If you're not familiar with the game, check out the rules here.
-      </a>
-    </p>
-
-    <p>
-      The rules are the same as in the above-linked article with some changes as follows to make the game a bit
-      simpler:
-    </p>
-
-    <ul>
-      <li>There will be 6 rounds or turns, each comprised of three dice rolls</li>
-      <li>Score combinations will be only from the lower section</li>
-      <li>No bonus scores of any kind</li>
-    </ul>
+    <main>
+      <div id="game">
```

Comment:
Since the game has a few different inputs and buttons, it might be a good idea to turn it into a form. I think the easiest way would be to change this `div` into a `form` element:

```suggestion
      <form id="game">
```